pcdsdevices/digital_signals.py

```diff
@@ -0,0 +1,20 @@
+from ophyd import Component as Cpt
+from ophyd import Device
+
+from .interface import BaseInterface
+from .signal import PytmcSignal
+
+
+class J120K(BaseInterface, Device):
+    """
+    A class representing the J120K 24V dry contact cooling flow switch.
+    """
+    flow_ok = Cpt(PytmcSignal, ':FSW:FLOW_OK', io='i',
+                  kind='normal', doc='flow rate nominal')
+
+    @property
+    def get_flow_ok(self):
+        """
+        returns True if flow rate is nominal
+        """
+        return self.flow_ok.get()
```

Comment:
I might cast this to a boolean, especially since your docstring says it returns True.  If you're going to have a getter here you may as well standardize the output
```suggestion
        return bool(self.flow_ok.get())
```

Reply:
thats `True`

Reply:
boolified